codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/smithy/generators/CargoTomlGenerator.kt

```diff
@@ -90,6 +92,7 @@ class CargoTomlGenerator(
                         "edition" to "2021",
                         "license" to moduleLicense,
                         "repository" to moduleRepository,
+                        "rust-version" to minimumSupportedRustVersion,
```

Comment:
**correctness**: I could be wrong but looking at this `minimumSupportedRustVersion` can be `null` here. Even though this is in a `listOfNotNull(...)` the type being added is a `Pair<String, String?>` which will never be `null`.

Reply:
Ah, good point. It surprisingly worked despite this, but I'll correct it.

Reply:
Fixed in f4dc57c92c3a0cf8b7cc1a0edfacc554f07ae631